tests/run-tests.sh

```
@@ -0,0 +1,359 @@
+        return 1
+    fi
+
+    # Check if file is not empty and has valid JSON
+    if [ ! -s "$auth_file" ]; then
+        return 1
+    fi
+
+    # Basic check for auth_token in the file
+    if grep -q "auth_token" "$auth_file" 2>/dev/null; then
+        # Verify the auth file contains cookies for this domain
+        local domain=$(echo "$url" | sed -E 's|https?://||' | sed -E 's|[:/].*||')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)